Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean

```diff
@@ -105,7 +108,7 @@ private theorem distinctPairs_increment :
 
 private lemma pairwiseDisjoint_distinctPairs :
     (P.parts.offDiag.attach : Set {x // x ∈ P.parts.offDiag}).PairwiseDisjoint
-      (distinctPairs G ε hP) := by
```

Comment:
Is this argument ordering desirable? I think it's likely that the original non-use of `variable`s was deliberate to affect the order.

Reply:
Oh nevermind, it's `private`

Reply:
I will also note that "original" here means "written by 2021 me", so I don't feel too bad for impeding on myself!